src/components/chart.svelte

```diff
@@ -91,6 +93,38 @@
         }
     }
 
+    function calculateMaxYValue(maxY, steps) {
+        if (maxY < 2) return Math.ceil(maxY)
+        const roundedToTen = Math.ceil(maxY / 10) * 10
+        return Math.ceil(roundedToTen / steps) * steps
+    }
+
+    function calculateMinYValue(minY, steps) {
```

Comment:
Merge these two functions into one and pass Math.floor and Math.ceil as param called `roundFunc`? I think that is valid